other-docs/guides/upgrading/v20.md

```diff
@@ -0,0 +1,87 @@
+---
+order: 20
+---
+
+# Upgrading to v20
+
+*If you are migrating from WordPress to Altis, check out the [migrating guide](../migrating/) first.*
+
+To upgrade to Altis v20, edit your `composer.json` and change the version constraint for `altis/altis` and any local environment
+modules to `^20.0.0`.
+
+```json
+{
+    "require": {
+        "altis/altis": "^20.0.0"
+    },
+    "require-dev": {
+        "altis/local-server": "^20.0.0"
+    },
+    "config": {
+        "platform": {
+            "php": "8.2"
+        }
+    }
+}
+```
+
+Once you have made these changes, run `composer update` and then run the `wp altis migrate` command:
+
+```sh
+# For cloud environments
+wp altis migrate
+
+# For local server
+composer server cli -- altis migrate
+```
+
+## Breaking Changes
+
+## Changes to the Altis platform
+
+Altis dev tools now come with wp-browser 4.x and Codeception 5.x This upgrade has some breaking changes. Please refer to
+the [wp-browser release notes](https://github.com/lucatume/wp-browser/releases/tag/4.0.0)
+and [Codeception release post ](https://codeception.com/07-28-2022/codeception-5.html)
+
+## PHP
+
+We have added experimental support for PHP 8.3. This is not yet recommended for production use but you can test your code with
+PHP 8.3 in local server
```

Comment:
I think "local server" should be "Local Server" for consistency with the other mentions in this file.